src/panels/profile/ha-long-lived-access-tokens-card.js

```diff
@@ -102,22 +106,25 @@ class HaLongLivedTokens extends LocalizeMixin(EventsMixin(PolymerElement)) {
   }
 
   async _handleCreate() {
-    const name = prompt(
-      this.localize("ui.panel.profile.long_lived_access_tokens.prompt_name")
-    );
+    const name = await showPromptDialog(this, {
+      text: this.localize(
+        "ui.panel.profile.long_lived_access_tokens.prompt_name"
+      ),
+    });
     if (!name) return;
     try {
       const token = await this.hass.callWS({
         type: "auth/long_lived_access_token",
         lifespan: 3650,
         client_name: name,
       });
-      prompt(
-        this.localize(
+      await showPromptDialog(this, {
```

Comment:
Should this be a prompt? The old alerts couldn't be copied, the new ones can I think?

Reply:
It can, but should it?
![image](https://user-images.githubusercontent.com/15093472/81221951-d8733b00-8fe3-11ea-9abc-61ed68858982.png)

Reply:
That looks bad 😄

Reply:
Way cleaner with prompt
![image](https://user-images.githubusercontent.com/15093472/81222307-60f1db80-8fe4-11ea-86fa-875e09b23002.png)